app/controllers/admin/accounting/transactions_controller.rb

```diff
@@ -27,15 +27,22 @@ def create
       # It's important we store the ID and type of the QB journal entry we just created
       # so that on the next sync, a duplicate is not created.
       @transaction.associate_with_qb_obj(journal_entry)
-
       @transaction.save!
+
+      # Create blank interest transaction. The interest calculator will pick this up and
+      # calculate the value, and sync it to quickbooks.
+      interest_transaction = ::Accounting::Transaction.find_or_initialize_by(transaction_interest_params
+        .merge(qb_transaction_type: ::Accounting::Transaction::LOAN_INTEREST_TYPE))
+      interest_transaction.save!
+
       flash[:notice] = t("admin.loans.transactions.create_success")
       render nothing: true
     rescue => ex
       # We don't need to display the message twice if it's a validation error.
       # But we do want to display the error if the QB API blows up.
       if ex.is_a?(ActiveRecord::RecordInvalid)
-        # Do nothing
+        # Only raise error if we had a problem saving the interest transaction
+        raise ex if @transaction.valid?
```

Comment:
You can do this more directly by checking if `ex.record == interest_transaction`, right? Then you wouldn't need the comment. You may have to make interest_transaction an instance var to do that, or initialize it to nil outside the begin block.